doc/nrf/installation/install_ncs.rst

```diff
@@ -611,16 +618,15 @@ To install the |NCS| system-wide, complete the following steps:
 Installation with Toolchain Manager
 ***********************************
 
-.. note::
+Toolchain Manager is a tool available from `nRF Connect for Desktop`_, a cross-platform tool that provides different applications that simplify installing the |NCS|.
+Both the tool and the application are available for Windows, Linux, and macOS.
 
+.. note::
    The Toolchain Manager installation is recommended only when using the nRF54H20 DK with the |NCS| v2.7.0.
-   When using any other DK, the Toolchain Manager installation is recommended for the |NCS| v1.9.x and earlier versions.
+   Since the release of |NCS| v2.0.0, the |nRFVSC| is the recommended IDE for manaaging the |NCS| Toolchain and working with the |NCS|.
```

Comment:
```suggestion
   Since the release of |NCS| v2.0.0, the |nRFVSC| is the recommended IDE for managing the |NCS| Toolchain and working with the |NCS|.
```